tests/test_scripts.py

```diff
@@ -68,3 +68,38 @@ def test_import_and_export(tmpdir, script_runner):
 
     command = 'pyhf json2xml {0:s} --specroot {1:s} --dataroot {1:s}'.format(temp.strpath,str(tmpdir))
     ret = script_runner.run(*shlex.split(command))
+    assert ret.success
```

Comment:
drive-by fix

Reply:
[pytest is still giving some errors in Python 2.7](https://travis-ci.org/diana-hep/pyhf/jobs/428334799#L510-L512)

```
>       assert ret.success
E       assert False
E        +  where False = <pytest_console_scripts.RunResult object at 0x7f80a9d50e50>.success
```

Reply:
something weird in the way the `pytest` file tmp file objects work in 2.7. They should be file-like but they are not. Changed it to `io.StringIO`

Reply:
see line 91